docs/DeveloperGuide.adoc

```diff
@@ -718,6 +732,52 @@ The added data security will assure the user that their data is safe and allows
 
 // end::dataencryption[]
 
+//tag::ai[]
+=== [Proposed] Artificial Intelligence (AI)
+Artificial intelligence (AI) is the simulation of human intelligence processes by machines, especially computer systems. These processes include learning (the acquisition of information and rules for using the information), reasoning (using rules to reach approximate or definite conclusions) and self-correction.
```

Comment:
Seems a bit too much background information, can consider shortening it

Reply:
The content has been reduced slightly.